### ISSUE TITLE
Question: Action to remove a property from an object?

### ISSUE THREAD

**Issue by mdgbayly** [davidkpiano/react-redux-form#128]
We've switched a lot of our internal models over from arrays of objects to normalized objects as we've found that gives more control with how we manage form state with RRF.  I know you are working on changes to track field state, but the normalized data seems to be a lot easier for us to manage.

However, one limitation we've hit is that currently RRF doesn't seem to provide a way to remove a property from an object. e.g. we have:

```
choices: {
    answer_1: {
        text: 'Happy',
        feedback: 'nice job'
    },
    answer_2: {
        text: 'Sad',
        feedback: 'oh dear'
    }
}
```

We need to be able to Remove an answer from the choices object:
e.g.
```
actions.remove('choices["answer_1"]);
```

```
choices: {
    answer_2: {
        text: 'Sad',
        feedback: 'oh dear'
    }
}
```

We've currently implemented this ourselves in our reducer.  I realise that icepick doesn't support dissocIn directly yet but we are using this [workaround](https://github.com/aearly/icepick/issues/14).

Let me know if you think this would be a useful thing for RRF and if so I can have a go at creating a PR.

**Comment by davidkpiano** [comment]
This would be a great addition. Currently, `actions.remove` is for arrays only, and the behavior should mirror https://lodash.com/docs#remove (which it currently doesn't; I'll fix that.)

Let's look to https://lodash.com/docs#omit for inspiration, so you can do:

```js
dispatch(actions.omit('choices', 'answer_1'))
```

What do you think?

**Comment by davidkpiano** [comment]
This is now available in `react-redux-form@0.9.13`! You can omit one or many properties at once:

```js
dispatch(actions.omit('choices', 'answer_1'));
dispatch(actions.omit('choices', ['answer_1', 'answer_3']));
```

Will close when the docs are updated.

**Comment by davidkpiano** [comment]
Now documented here: https://davidkpiano.gitbooks.io/react-redux-form/content/model_actions.html#actionsomitmodel-props
